packages/components/src/slot-fill/index.tsx

```diff
@@ -84,17 +84,12 @@ export function createSlotFill( key: SlotKey ) {
 		props: DistributiveOmit< SlotComponentProps, 'name' >
 	) => <Slot name={ key } { ...props } />;
 	SlotComponent.displayName = `${ baseName }Slot`;
+	// deprecated legacy property, should use `slotFill.name` instead of `slotFill.Slot.__unstableName`
```

Comment:
This will make it show up in IntelliSense when accessed.

```suggestion
	/** @deprecated Use `slotFill.name` instead of `slotFill.Slot.__unstableName`. */
```